common/selectors/wallet.js

```diff
@@ -39,3 +40,25 @@ export function getTokenBalances(state: State): TokenBalance[] {
     custom: t.custom
   }));
 }
+
+export function getTxFromTransactionsBySignedTx(
+  state: State,
+  signedTx: string
+): ?BroadcastStatusTransaction {
+  const transactions = state.wallet.transactions;
+  const matchingTxs = transactions.filter(function(obj) {
+    return obj.signedTx === signedTx;
+  });
+  return matchingTxs ? matchingTxs[0] : null;
```

Comment:
I would just reuse `getTxFromBroadcastStatusTransactions` in here.

Reply:
agreed.